manifests/init.pp

```diff
@@ -20,6 +20,9 @@
   Optional[Integer] $metrics_server_port      = undef,
   Optional[String]  $metrics_server_db_name   = undef,
   Optional[String]  $override_metrics_command = undef,
+  Optional[Array[String]] $puppetserver_excludes = ['pe-puppet-profiler', 'file-sync-storage-service'],
```

Comment:
We could also exclude the `pe-master`, and `pe-jruby-metrics` hashes as well. They're not nearly as big as `pe-puppet-profiler`, but worth excluding as well.

Also, this should default to just File Sync, `['file-sync-storage-service']`, if the PE version is older than 2017.3 --- as the `pe-` metrics were the only source of the profiling data prior to Puppet Server 5.0.

Reply:
Good call. I'll put in some logic to determine the defaults and allow for future version changes.

Reply:
Updated in the recent commits.